code/LINDA/LINDA_fire.dm

```diff
@@ -152,18 +152,18 @@
 // Garbage collect itself by nulling reference to it
 
 /obj/effect/hotspot/proc/Kill()
-	air_master.hotspots -= src
-	DestroyTurf()
-	qdel(src)
+	PlaceInPool(src)
```

Comment:
PlaceInPool specifically calls Destroy() before resetting all the variables back to default.